python/cudf/cudf/core/column/column.py

```diff
@@ -146,7 +146,9 @@ def data_array_view(
             obj = None
         return cuda.as_cuda_array(obj).view(self.dtype)
 
-    def mask_array_view(self, mode="read") -> "cuda.devicearray.DeviceNDArray":
+    def mask_array_view(
+        self, *, mode="read"
```

Comment:
```suggestion
        self, *, mode="write"
```

This one has still snuck through :)

Reply:
Yikes, Sorry!